.github/workflows/dapr.yml

```diff
@@ -146,6 +146,18 @@ jobs:
         run: |
           echo "Build docker image and push image..."
           make docker-push TARGET_OS=${{ matrix.target_os }} TARGET_ARCH=${{ matrix.target_arch }} DAPR_REGISTRY=${{ secrets.DOCKER_REGISTRY }} DAPR_TAG=${{ env.REL_VERSION }}
+      - name: GitHub container registry login
+        if: matrix.target_os != 'darwin' && github.event_name != 'pull_request'
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
```

Comment:
`github.actor` is the username of the user that initiated the workflow run.

This would differ based on who is triggering the workflow.

Should we not hardcode this is to be dapr / dapr bot, or maybe `github.repository_owner` ? I'm unclear on the correct value, but I don't think it is `github.actor`.

Also the secret we have today in our organizational secret must have the right scope to read / write packages. I am not sure whether it does. We may need to roll the organizational secret - but we also have to double check which user this secret is scoped to.

Thoughts?

Reply:
@pravinpushkar due to safety concerns, please post a PR in your fork repo, then we can see the result here. Then we can pick one maintainer who has permission to use secrets to finish this job.

/cc @yaron2 @artursouza WDYT?

Reply:
@berndverst I just verified `github.repository_owner` is also working and looks to be a better option than `github.actor`.
By default `secrets.GITHUB_TOKEN` has read/write package scope.

As of now I am changing the username to be `github.repository_owner` and will wait for further feedback.

Reply:
@daixiang0 I created one PR in my fork (https://github.com/pravinpushkar/dapr/runs/5106183840?check_suite_focus=true) .. but publish did not run I guess because of `if: github.event_name != 'pull_request'` under publish in yml file. I will branch out from this recreate the fork PR.

Reply:
I ran the workflow in my fork PR but it is failing with this reason - `docker manifest create ***/dapr:edge ***/dapr:edge-linux-amd64 ***/dapr:edge-linux-arm ***/dapr:edge-linux-arm64 ***/dapr:edge-windows-amd64
no such manifest: ***/dapr:edge-linux-amd64`
here is the link for the ran action - https://github.com/pravinpushkar/dapr/runs/5122070859?check_suite_focus=true

@artursouza  Any thoughts on this?

Reply:
@pravinpushkar I believe it's caused by `${{ secrets.DOCKER_REGISTRY }}` not configured in your repository.